packages/reakit/src/Disclosure/DisclosureState.ts

```diff
@@ -98,7 +98,20 @@ export function useDisclosureState(
   React.useEffect(() => {
     if (typeof animated === "number" && animating) {
       setTimeout(() => setAnimating(false), animated);
+      return () => {};
     }
+    if (animated && animating && process.env.NODE_ENV === "development") {
+      const timeout = setInterval(() => {
+        console.warn(
+          "[Reakit] It's been 8 seconds but stopAnimation has not been called. Does the discolusure element have a CSS transition?"
+        );
+        setAnimating(false);
```

Comment:
I don't think we should call `setAnimating` here, since this would cause a different behavior between client and server. We can just warn and let the user decide what to do.